features/deeplearning/src/main/java/org/opennms/features/deeplearning/shell/LoadModel.java

```diff
@@ -33,21 +33,25 @@
 import org.apache.karaf.shell.api.action.Command;
 import org.apache.karaf.shell.api.action.lifecycle.Service;
 import org.opennms.alec.engine.deeplearning.TFModel;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 @Command(scope = "opennms-alec", name = "tensorflow-load-model", description = "Validate that the TensorFlow model at the given path can be loaded.")
 @Service
 public class LoadModel implements Action {
 
+    private static final Logger LOG = LoggerFactory.getLogger(LoadModel.class);
+
     @Argument(name="model path", required = true)
     private String modelPath;
 
     @Override
     public Object execute() {
         try (TFModel tfModel = new TFModel(modelPath)) {
-            System.out.println("Model successfully loaded: " + modelPath);
+            LOG.info("Model successfully loaded: {}",  modelPath);
```

Comment:
We use `System.out` intentionally in Karaf shell commands so that they show up in console output. LOGs aren't ideal here.